src/probnum/kernels/_rational_quadratic.py

```diff
@@ -0,0 +1,85 @@
+"""Rational quadratic kernel."""
+
+from typing import Optional
+
+import numpy as np
+import scipy.spatial.distance
+
+import probnum.utils as _utils
+from probnum.type import IntArgType, ScalarArgType
+
+from ._kernel import Kernel
+
+_InputType = np.ndarray
+
+
+class RatQuad(Kernel[_InputType]):
+    """Rational quadratic kernel.
+
+    Covariance function defined by :math:`k(x_0, x_1) = \\big(1 + \\frac{\\lVert x_0 -
+    x_1 \\rVert^2}{2\\alpha l^2}\\big)^{-\\alpha}`, where :math:`\\alpha > 0`. For
+    :math:`\\alpha \\rightarrow \\infty` the rational quadratic kernel converges to the
+    :class:`~probnum.kernels.ExpQuad` kernel.
+
+    Parameters
+    ----------
+    input_dim :
+        Input dimension of the kernel.
+    lengthscale :
+        Lengthscale of the kernel. Describes the input scale on which the process
+        varies.
+    alpha :
+        Scale mixture. Positive constant determining the weighting between different
```

Comment:
any particular reason for calling this `alpha` and not `scale_mixture` or something along those lines?

Reply:
Mostly I stuck to conventions set by `gpytorch` and `gpy`. I think for kernels some of these parameters are so set in stone, that they are more recognizable by their greek letters.

Reply:
But isn't it a bit odd that we have word-names for everything (lengthscale, etc.) but greek letters nu and alpha? In any case, I am okay with it, but just wanted to voice thisopinion at some point...